interfaces/apparmor/apparmor.go

```diff
@@ -37,46 +37,69 @@ import (
 	"github.com/snapcore/snapd/osutil"
 )
 
-// LoadProfile loads an apparmor profile from the given file.
+// LoadProfile loads apparmor profiles from the given files.
 //
 // If no such profile was previously loaded then it is simply added to the kernel.
 // If there was a profile with the same name before, that profile is replaced.
-func LoadProfile(fname string) error {
-	return loadProfile(fname, dirs.AppArmorCacheDir)
+func LoadProfiles(fnames []string) error {
```

Comment:
Suggestion, use `LoadManyProfiles` and perhaps use varadic arguments (less annoying in testing)